test/relationTests.js

```diff
@@ -36,6 +36,38 @@ var RoleLinkMockup = nohm.model('RoleLinkMockup', {
   },
   idGenerator: 'increment'
 });
+var UserWithUuidLinkMockup = nohm.model('UserWithUuidLinkMockup', {
+  properties: {
+    name: {
+      type: 'string',
+      defaultValue: 'UUID_user'
+    }
+  },
+  idGenerator: function (cb) {
+    function s4() {
+      return Math.floor((1 + Math.random()) * 0x10000)
+        .toString(16)
+        .substring(1);
+    }
+    cb(s4() + s4() + '-' + s4() + '-' + s4() + '-' + s4() + '-' + s4() + s4() + s4());
```

Comment:
This seems overly complicated for a test. You can leave it, if changing this to something simpler seems like too much work - it's not important.